CMakeLists.txt

```diff
@@ -3,6 +3,10 @@
 # for further info why
 cmake_minimum_required(VERSION 3.16.3 FATAL_ERROR)
 
+if(CMAKE_VERSION VERSION_GREATER_EQUAL "3.30.0")
```

Comment:
I think the normal CMake way is to check if the policy exists, rather than a specific version of cMake -- see the checks immediately below for examples.